.env.sample

```diff
@@ -22,6 +22,8 @@
 # The default cache expiration time in seconds if none is set
 # (default=60)
 #EXPIRATION_TIME_DEFAULT_SECONDS
+#CONTRACT_NOT_FOUND_ERROR_TTL_SECONDS
+#TOKEN_NOT_FOUND_ERROR_TTL_SECONDS
```

Comment:
This is missing what the envar represents.

Reply:
Added in [f64c4da](https://github.com/safe-global/safe-client-gateway-nest/pull/578/commits/f64c4dad0e26f2530a411001a900ee3b255cc940)